bin/dlang/install

```diff
@@ -14,7 +14,14 @@ case "$DMD" in
     dmd*   ) PKG= ;;
     1.*    ) PKG="dmd1=$DMD-$DIST" ;;
     2.*.s* ) PKG="dmd-transitional=$DMD-$DIST" ;;
-    2.*    ) PKG="dmd-bin=$DMD libphobos2-dev=$DMD" ;;
+    2.*    ) # For some reason &>/dev/null at the next line would result at
+             # always-success status code:
```

Comment:
`&>` is a *bashism* and this is a `sh` script. But why did you wanted to use `&>/dev/null` and where exactly?

Reply:
Oops, this is comment for `apt-cache` based version which I forgot to remove :D

Reply:
Updated